src/BlockTypes/MiniCart.php

```diff
@@ -74,6 +104,26 @@ protected function enqueue_data( array $attributes = [] ) {
 
 		parent::enqueue_data( $attributes );
 
+		$label_info = $this->get_tax_label();
+
+		$label_including_tax               = $label_info['label_including_tax'];
+		$display_cart_prices_including_tax = $label_info['display_cart_prices_including_tax'];
+
+		$this->label_including_tax               = $label_including_tax;
+		$this->display_cart_prices_including_tax = $display_cart_prices_including_tax;
+
+		$this->asset_data_registry->add(
+			'labelIncludingTax',
+			$this->label_including_tax,
+			''
+		);
+
+		$this->asset_data_registry->add(
+			'displayCartPricesIncludingTax',
+			$this->display_cart_prices_including_tax,
+			false
+		);
```

Comment:
This is also producing an error for me:

> Fatal error: Uncaught Exception: Overriding existing data with an already registered key is not allowed
in /wp-content/plugins/woocommerce-gutenberg-products-block/src/Assets/AssetDataRegistry.php on line 380

Reply:
Now it should be fine!